### DISCUSSION TITLE
Global playlist

### DISCUSSION THREAD

**Discussion by vbabiy** [mediacms-io/mediacms#742] [Feature Requests]
Is there a way to have a global playlist? I have a collection of videos that users need to watch in order?

I was trying to do this with a playlist with the playlist are user-centric.

Thanks for the effort you have put into media CMS

**Comment by mgogoulos** [comment]
Hi, you can share the URL of a playlist, and if videos are public on that playlist (not private) they will be able to play them. 
Or perhaps you mean that a page with all available playlists would be nice to have?

**Comment by vbabiy** [comment]
Yeah I was thinking of having a page that lists them.

**Comment by morbificagent** [comment]
This would be e a nice Feature!

+1 here

**Comment by tdsotm** [comment]
I was also surprised to see that after creating 20 playlists, they are only visible on my account. Sharing the link would work as a workaround but it's not desirable

**Comment by cdorsat** [comment]
Is it possible to shuffle playlists on the homepage?  And not just the latest movies.
